charts/sn-platform/values.yaml

```diff
@@ -397,6 +397,17 @@ certs:
     enabled: false
     component: public-cert-issuer
     type: acme
+
+  istio_internal_issuer:
```

Comment:
Is it possible to just reuse the existing `internal_issuer` and `public_issuer`?

Reply:
If we change public_issuer and internal_issuer from `Issuer` to `ClusterIssuer`, it will be easy to reuse.

Reply:
I see. We generally prefer namespace-scoped resources over cluster-scoped resources, so it's fine to create separate issuers here.